api/app/src/main/kotlin/packit/controllers/PacketController.kt

```diff
@@ -69,8 +70,11 @@ class PacketController(private val packetService: PacketService)
         @RequestParam paths: List<String>,
         response: HttpServletResponse
     ) {
-        response.contentType = "application/zip"
-        response.setHeader("Content-Disposition", "attachment; filename=$id.zip")
+        val headers = HttpHeaders().apply {
+            contentType = MediaType.valueOf("application/zip")
+            contentDisposition = ContentDisposition.parse("attachment; filename=$id.zip")
+        }
+        headers.map { response.setHeader(it.key, it.value.first()) }
```

Comment:
What was the motivation behind this change? Creating a headers map just to iterate over it afterwards seems more complicated than the existing version.

Reply:
As far as possible, I wanted to make use of the libraries written by other people, that ensure the strings are parsed as valid http headers, rather than pass any old string into `response.setHeader`. Underlyingly, `contentType = ` makes use of `HttpHeaders.setContentType` (which performs a few validation checks).

We can instead use `response.contentType = `.

The other reason is the typing of `response.setHeader` makes it pretty ugly with `toString` everywhere. The full change would read as:

```kt
response.contentType = MediaType.valueOf(mediaType.toString()).toString()
response.setHeader("Content-Disposition", ContentDisposition.attachment().filename("$id.zip").build().toString())
```